dialogue-clients/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -0,0 +1,10 @@
+options:
+  javaPackage: com.palantir.dialogue.clients
+  javaVisibility: packagePrivate
+namespaces:
+  client.dns:
+    docs: Dialogue DNS metrics.
+    metrics:
+      tasks:
+        type: counter
+        docs: Number of active Dialogue DNS update background tasks currently scheduled.
```

Comment:
We're using this metric in the integration test as we're no longer able to query individual threads, because there's no longer a 1-1 mapping with threads, and the executor threads will stay alive for some time without incoming tasks awaiting reuse.